pkg/ctl/cmdutils/nodegroup_filter_test.go

```diff
@@ -0,0 +1,126 @@
+package cmdutils_test
```

Comment:
As discussed offline, we might want to do some mixed public/private unit tests, but for now public-only tests remain conventional...